### DISCUSSION TITLE
[Script request] Update Trilium (in maintenance mode) in to newer TriliumNext

### DISCUSSION THREAD

**Discussion by Lewis8265** [community-scripts/ProxmoxVE#680] [Request script] [answered]
### Application Name

TriliumNext

### Website

https://github.com/TriliumNext/Notes

### Description

Trilium is currently available here: https://community-scripts.github.io/ProxmoxVE/scripts?id=trilium

It is in maintenance mode and will not recieve updates, meanwhile the newer script TriliumNext was updated only last week and is looking like it will be the way forward with maintenance updates:

https://github.com/TriliumNext/Notes

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by jsapede** [comment]
moreover, beta android application is now available ! 
https://github.com/TriliumNext/Notes/issues/72#issuecomment-2517011699

**Reply by Lewis8265** [reply]
Ooo nice, progress!

**Comment by knario** [comment, marked as answer]
Hi, from the script it automatically updates to trilium Next.
I have been using trilium since the script was created and I am currently on the latest version of trilium Next.
Have you tried upgrading?

**Reply by Lewis8265** [reply]
Indeed it does, thanks. I have no idea if it was Trilium Next was originally installed as I couldn't find a version number, but I ran the command in the Console of the container as if I was updating it, where it says it is updated to 0.90.12. This matches Trilium Next. The Trilium in maintenance mode is 0.63.7. The team should update the website to reflect that it is Trilium Next, thanks again!
